sympy/combinatorics/fp_groups.py

```diff
@@ -738,19 +738,29 @@ def first_in_class(C, Y=[]):
 #                    Simplifying Presentation
 #========================================================================
 
-def simplify_presentation(*args):
+def simplify_presentation(*args, **kwargs):
     '''
     For an instance of `FpGroup`, return a simplified isomorphic copy of
     the group (e.g. remove redundant generators or relators). Alternatively,
     a list of generators and relators can be passed in which case the
     simplified lists will be returned.
 
+    By default, the generators of the group are unchanged. If you would
+    like to remove redundant generators, set the keyword argument
+    `change_gens = True`.
+
     '''
+    change_gens = False
+    for k, v in kwargs.items():
+        if k == "change_gens":
+            change_gens = v
```

Comment:
This is the recommended way of dealing with keywords that need not exist:

    change_gens = kwargs.get('change_gens', False)

Reply:
I was looking for something like that, thanks.